scripts/benchmarks/benchmark.py

```diff
@@ -220,6 +227,8 @@ def prepare_dataset(
 
         # call the map
         ds = self.dataset_split.map(format_fn, **kwargs)
+        # set an attribute to indicate dataset has already been tokenized
+        self.is_pretokenized = 'input_ids' in ds.column_names and 'labels' in ds.column_names
```

Comment:
all this can be avoided if you just use a new `defaults.yaml` no?

Reply:
ok so the strategy now is to just use the existing null check to undo some default settings